kloud51/templates/menu/language_chooser.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n menu_tags %}
+{# TODO: Let's have the Language (country) flags as well. #}
+
+<li class="dropdown">
+  <a class="dropdown-toggle" data-toggle="dropdown" href="#" id="Language" aria-expanded="false">
+    {{ current_language }} <span class="caret"></span>
+  </a>
+  <ul class="dropdown-menu">
+    {% for language in languages %}
+      <li class="lang{% if current_language == language.0 %} active{% endif %}">
+        <a href="{% page_language_url language.0 %}" title="{% trans "Change to language:" %} {{ language.1 }}">{{ language.1 }}</a>
```

Comment:
@AmirSavand this is where you'd include the language field flag.

Reply:
ok